addons/sales_team/static/src/js/crm_team_form.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { formView } from "@web/views/form/form_view";
+import { FormController } from "@web/views/form/form_controller";
+import { user } from "@web/core/user";
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+
+
+class CrmTeamFormController extends FormController {
```

Comment:
Would it be possible to write a small test for that ? That way if someday a framework team changes something, it is tested :disguised_face: